style.css

```diff
@@ -90,3 +96,576 @@ p {
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.history3 .date {
+  width: 29px;
+  height: 16px;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.paragraph3 {
+  width: 295px;
+  height: 76px;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  padding-top: 20px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.grid-container4 {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 5px;
+  background-color: #fff;
+  width: 327px;
+  height: 596px;
+  border: 1px solid #dfe1e6;
+  border-radius: 16px;
+}
+
+.w-image4 {
+  padding-top: 15px;
+  width: 295px;
+  height: 220px;
+  border-radius: 8px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.m-post2 {
+  width: 295px;
+  height: 36px;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 32px;
+  line-height: 44px;
+  color: #172b4d;
+  padding-top: 10px;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.history4 {
+  padding-top: 70px;
+}
+
+.history4 .canopy {
+  width: 55px;
+  height: 16px;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #344563;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.history4 .spa {
+  width: 88px;
+  height: 16px;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.history4 .date {
+  width: 29px;
+  height: 16px;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 600;
+  font-size: 13px;
+  line-height: 16px;
+  color: #7a869a;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+}
+
+.paragraph4 {
+  width: 295px;
+  height: 76px;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #344563;
+  padding-top: 20px;
+  flex: none;
+  order: 1;
+  flex-grow: 0;
+}
+
+.card {
+  width: 375px;
+  height: 1320px;
+  left: 375px;
+  background-color: #fff;
+  border-radius: 0 80px 0 0;
+}
+
+.p-about {
+  padding-top: 10px;
+}
+
+.resume {
+  border-radius: 8px;
+  background-color: #fff;
+  padding: 10px;
+  border: 2px solid #6070ff;
+  color: #396df2;
+  font-family: 'poppins', sans-serif;
+  font-style: normal;
+  font-size: 17px;
+  font-weight: 500;
+  line-height: 24px;
+  margin-top: 28px;
+  margin-bottom: 32px;
+  margin-left: 25px;
+}
+
+.resume:hover {
+  background-color: #396df2;
+  color: #fff;
+}
+
+.resume:active {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.resume:disabled {
+  background-color: #fff;
+  color: #7a869a;
+}
```

Comment:
- [ ] Kindly make sure all the buttons have the interactions styles setup `hover, active, disable`